modules/n1ql/pages/n1ql-rest-api/admin.adoc

```diff
@@ -66,7 +66,7 @@ Returns information about all clusters.
 
 ==== Security
 
-[options="header", cols=".^3,.^4"]
+[options="header", cols=".^3a,.^4a"]
```

Comment:
@simon-dew Similarly, can you put an explanation for this in a comment in the file?

Reply:
I'm going to push back on this one, because authors should not be editing this file directly. If future authors don't know what any particular AsciiDoc markup is for, they should refer to the [AsciiDoctor User Manual](https://asciidoctor.org/docs/user-manual/).